cabby/rvs/templates.py

```diff
@@ -323,11 +334,15 @@ def add_entity_span(entity_tag: str, instruction: str) -> Dict[str, Tuple[int, i
     Returns:
       A dictionary of entities (keys) and spans (values).
   '''
-  pattern = re.compile(entity_tag, re.IGNORECASE)
+
+  keyword_processor = flashtext.KeywordProcessor(case_sensitive=False)
+  keyword_processor.add_keyword(entity_tag)
+
+  keywords_found = keyword_processor.extract_keywords(instruction, span_info=True)
 
   entities_span_dict = {}
-  for m in pattern.finditer(instruction):
-    entities_span_dict[m.group()] = (m.start(), m.end())
+  for m in keywords_found:
```

Comment:
Please use a more descriptive variable than 'm'.

Reply:
Changed to keyword_found

Reply:
It still says "for *m* in keywords_found:"... Why is it 'm'?

Reply:
Checked, and now changes appear.